go/vt/vttablet/tabletserver/throttle/check.go

```diff
@@ -148,20 +148,19 @@ func (check *ThrottlerCheck) Check(ctx context.Context, appName string, storeTyp
 	}
 
 	checkResult = check.checkAppMetricResult(ctx, appName, storeType, storeName, metricResultFunc, flags)
-	check.throttler.lastCheckTimeNano.Store(time.Now().UnixNano())
```

Comment:
Removed from `check.,go`. Delegated to, and consolidated in `checkStore()` in `throttler.go`.